src/pages/Environments.js

```diff
@@ -133,36 +294,41 @@ const Environments = () => {
   const [sort, setSort] = useState({ field: 'project', direction: 'asc' })
   const [diskSort, setDiskSort] = useState({ field: 'project', direction: 'asc' })
   const [shouldFilterRuntimesByCreator, setShouldFilterRuntimesByCreator] = useState(true)
+  const [migrateDisk, setMigrateDisk] = useState()
 
   const refreshData = Utils.withBusyState(setLoading, async () => {
+    await refreshWorkspaces()
     const creator = getUser().email
 
+    const getWorkspace = (ws => (namespace, name) => _.get(`${namespace}.${name}`, ws))(
+      getWorkspaces()
+    )
```

Comment:
This took me a minute to figure out what it was doing. Is there a need to capture the return value of `getWorkspaces` like this. It's returning the value of a ref, so it wouldn't be expensive to call.

```suggestion
    const getWorkspace = (namespace, name) => _.get(`${namespace}.${name}`, getWorkspaces())
```

Reply:
I guess not, I was just eliminating an extra function call each time `getWorkspace` is called.

Reply:
Sure, but since `getWorkspaces` is only reading a value from a ref object, I'd claim that the optimization isn't worth the loss in readability.
https://github.com/DataBiosphere/terra-ui/blob/56d4157468bab113ae025880c4d1dcce513d30f7/src/libs/react-utils.js#L30-L34